components/brave_wallet/browser/hd_keyring.h

```diff
@@ -48,7 +48,7 @@ class HDKeyring {
   std::string GetDiscoveryAddress(size_t index) const;
   // Find private key by address (it would be hex or base58 depends on
   // underlying hd key
-  std::string GetEncodedPrivateKey(const std::string& address);
+  virtual std::string GetExportEncodedPrivateKey(const std::string& address);
```

Comment:
Rename doesn't seem very necessary to me, and we're missing a few places in the solana keyring unit tests that still using `GetEncodedPrivateKey`.
Maybe easier to just keep the original name to minimize the changes for uplift.